components/classroom/project.js

```diff
@@ -0,0 +1,16 @@
+// function extracts projectID from URL of the page
+function getProjectIDFromURL() {
```

Comment:
Function is readable and clean. Maybe include a few comments describing what some functions do. For example, 'window.location.search', 'URLSearchParams', etc.